src/src/WebServer/Metrics.cpp

```diff
@@ -0,0 +1,121 @@
+# include "../WebServer/Metrics.h"
+# include "../WebServer/WebServer.h"
+# include "../../ESPEasy-Globals.h"
+# include "../Commands/Diagnostic.h"
+# include "../ESPEasyCore/ESPEasyNetwork.h"
+# include "../ESPEasyCore/ESPEasyWifi.h"
+# include "../../_Plugin_Helper.h"
+# include "../Helpers/ESPEasyStatistics.h"
+# include "../Static/WebStaticData.h"
+HELPERS_ESPEASY_MATH_H
+#ifdef ESP32
+# include <esp_partition.h>
+#endif // ifdef ESP32
+
+
+void handle_metrics() {
```

Comment:
In `Metrics.h` these functions are conditionally compiled (`#ifdef WEBSERVER_METRICS`), shouldn't that be here as well, or does the linker take care of leaving it out of the final binary? (I'd suggest to add the `#ifdef` here too)

Reply:
Ah missed that one?
The linker will remove unused functions (as long as they are not part of a class), so it won't matter here but it is good practice to have it wrapped too.